src/app/app.component.ts

```diff
@@ -31,12 +31,16 @@ export class AppComponent implements OnInit {
     ).subscribe(
       (resource: ReadResource) => {
         this.testthing = resource;
-        console.log(this.testthing);
-        this.testValue = this.testthing.getValues('http://0.0.0.0:3333/ontology/0001/anything/v2#hasUri')[0];
+        // console.log(this.testthing);
+        this.testValue = this.testthing.getValues('http://0.0.0.0:3333/ontology/0001/anything/v2#hasInterval')[0];
       }
     );
 
   }
 
+  ngAfterViewInit(): void {
+    // console.log(this);
+  }
+
 
```

Comment:
left over debugging code that can be removed as well I think

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/26/commits/59066f2762b9a001a2eefd39e6155aac23f34bd0